app/models/band.rb

```diff
@@ -6,10 +6,21 @@ class Band < ActiveRecord::Base
   scope :starts_with, -> (name) { where('name like ?', "%#{name}%") }
   
   has_many :musicians
+  has_many :youtube_links
   has_and_belongs_to_many :event_types
   has_and_belongs_to_many :genres
 
   has_one :user
 
+  has_many :youtube_links
```

Comment:
That's because bands have many many YouTube links :joy: 

PD: I'll get it sorted once I get to the office :stuck_out_tongue_closed_eyes: